src/source/SwiftGenerator.scala

```diff
@@ -0,0 +1,59 @@
+/**
+  * Copyright 2016 Dropbox, Inc.
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *    http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+package djinni
+
+import java.io.{File, FileWriter, BufferedWriter}
+
+import djinni.ast._
+import djinni.generatorTools._
+
+class SwiftGenerator(spec: Spec) extends Generator(spec) {
```

Comment:
This should maybe be called SwiftBridgeGenerator or something similar?  It should be clear that it doesn't actually generate Swift code, which is something which might be added in future (if someone tries to create a more Swifty interface).